### ISSUE TITLE
Carousel slide direction

### ISSUE THREAD

**Issue by preventsenior** [twbs/bootstrap#15634]
It would be nice to set up the direction of transition, for example: right, left, up, down.

**Comment by shenxiuqiang** [comment]
_glyphicons.scss  error
.glyphicon-eur                    { &:before { content: \20ac; } } 
 should is
.glyphicon-eur                    { &:before { content: "\20ac"; } }

and other

**Comment by juthilo** [comment]
@shenxiuqiang That's entirely unrelated to this issue. Please [open a new issue (in our Sass repo)](https://github.com/twbs/bootstrap-sass/issues/new) with more details about which type and version of bootstrap-sass you're using. Thanks!

**Comment by kkirsche** [comment]
This should be relatively simple to add by modifying this part:
https://github.com/twbs/bootstrap/blob/master/js/carousel.js#L121


@mdo / @cvrebert is this something you'd be interested in seeing a PR for or wait till V4 to add if it isn't in it when it's released?

**Comment by mdo** [comment]
Yeah, not until v4, @kkirsche.

**Comment by cvrebert** [comment]
Added this to our internal "consider for v4" tracking list.
